classes/item.rb

```diff
@@ -1,7 +1,8 @@
 require 'date'
+require_relative './author'
```

Comment:
Nice, great!!
Also I forgot to mention, u don't need this [line](https://github.com/Ibizugbe/catalog_of_my_things/pull/39/files#diff-de9d57b5bbe8fc257bd084ba1277ca73c051189be68c5fbb302c0cdbd4fcd731R2) here and it also breaks the entire code